src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.service.TransactionReaderService;
+import core.basesyntax.service.TransactionWriterService;
+import core.basesyntax.service.impl.FruitServiceImpl;
+import core.basesyntax.service.impl.TransactionReaderServiceImpl;
+import core.basesyntax.service.impl.TransactionWriterServiceImpl;
+import core.basesyntax.strategy.TransactionStrategy;
+import java.util.List;
+
+public class Main {
+    private static TransactionReaderService transactionReaderService =
+            new TransactionReaderServiceImpl();
+    private static FruitService fruitService =
+            new FruitServiceImpl();
+    private static TransactionStrategy transactionStrategy =
+            new TransactionStrategy();
+    private static FruitStorage fruitStorage =
+            new FruitStorage();
+    private static TransactionWriterService transactionWriterService =
+            new TransactionWriterServiceImpl();
+
+    public static void main(String[] args) {

```

Comment:
See main method structure. We should have 5 method calls here (besides objects initialization if needed)
Remove for loop please
![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/2199c7fb-8a66-45d7-ada2-0795d4eb8d29)

![image](https://github.com/mate-academy/jv-fruit-shop/assets/20326858/56a96cac-cf46-4956-909b-b778ccea2717)